R/stat-bin.r

```diff
@@ -111,9 +113,10 @@ StatBin <- ggproto("StatBin", Stat,
       stop("Only one of `boundary` and `center` may be specified.", call. = FALSE)
     }
 
-    if (is.null(params$breaks) && is.null(params$binwidth) && is.null(params$bins)) {
+    if (is.null(params$breaks) && (is.null(params$binwidth) || !(is.numeric(params$binwidth) || is.function(params$binwidth))) && is.null(params$bins)) {
```

Comment:
Why does this need to be changed? If a function is supplied `binwidth` will no longer be `NULL`